firmwares/piezo_i2c_backpack.ino

```diff
@@ -81,18 +81,23 @@ void onReceive(int howMany) {
     Serial.println(ms);
     Serial.println("--------------------");
   #endif
-
+  Serial.println(OUTPUT);
   pinMode(pin, OUTPUT);
 
   if (command == NO_TONE) {
     noTone(pin);
   }
 
+  // delay between notes
+  int pauseBetweenNotes = ms * 1.3;
+  delay(pauseBetweenNotes);
+  noTone(pin);
```

Comment:
So I looked at the mario script and my old piezo scripts and to keep the tempo, I think you'd always have it even if there was no tone?